portfolio/src/main/webapp/comments.js

```diff
@@ -21,6 +25,9 @@ async function fetchComments() {
     addDeleteButton();
     document.getElementById('no-comments').remove();
   }
+  // update page
+  updateReplyFromAccessBasedOnAuthState();
```

Comment:
Is there a reason, we want to create elements and hide them based on authstate? 
Why do we not wanna do the otherway. I.e create elements only when logged in?

Reply:
well, this particular form is hardcoded in html. To fix this, I need to dynamically create it. I agree, your approach seems more logical